src/NzbDrone.Core/Tv/AddSeriesService.cs

```diff
@@ -67,7 +69,13 @@ public List<Series> AddSeries(List<Series> newSeries, bool ignoreErrors = false)
                     var series = AddSkyhookData(s);
                     series = SetPropertiesAndValidate(series);
                     series.Added = added;
-                    seriesToAdd.Add(series);
+                    if (existingSeries.Any(f => f.TvdbId == series.TvdbId) || seriesToAdd.Any(f => f.TitleSlug == series.TitleSlug) || seriesToAdd.Any(f => f.TvdbId == series.TvdbId))                    {
```

Comment:
Do these in separate if blocks. Something like.

```
if (existingSeries.Any(f => f.TvdbId == series.TvdbId))
{
  _logger.Debug("TVDB ID {0} was not added due to validation failure: Series already exists", s.TvdbId);
  continue;
}

var duplicateSlug = seriesToAdd.FirstOrDefault(f => f.TitleSlug == series.TitleSlug)

if (duplicateSlug != null)
{
  _logger.Debug("TVDB ID {0} was not added due to validation failure: Duplicate slug {1} used by series {2}", s.TvdbId, s.TitleSlug, duplicateSlug.TvdbId);
  continue;
}
...
```

(also fixes the spelling of encountered.